repl/src/main/scala/com/cloudera/livy/repl/Session.scala

```diff
@@ -140,7 +157,7 @@ class Session(interpreter: Interpreter)
         (EVALUE -> e.getMessage) ~
         (TRACEBACK -> List())
     }
+
+    compact(render(resultInJson))
```

Comment:
We know for sure Kryo can serialize strings, but might not be able to serialize non POJOs.